pkgs/applications/graphics/kcc/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "KindleComicConverter";
```

Comment:
Should not be different from the attribute in `top-level.nix`.
I'm divided on what to use here. On GitHub they use `kcc` but on PyPi they use this...

Reply:
Plus we don't use uppercase in package names (https://nixos.org/nixpkgs/manual/#sec-package-naming)
So it's either `kcc` or `kindlecomicconverter`